metadata/schema_test.go

```diff
@@ -117,16 +143,16 @@ func TestBuildArraySchema(t *testing.T) {
 	var err error
 
 	zeroArr := [0]int{}
-	schema, err = buildArraySchema(reflect.ValueOf(zeroArr), nil)
+	schema, err = buildArraySchema(reflect.ValueOf(zeroArr), nil, false)
 	assert.Equal(t, errors.New("Arrays must have length greater than 0"), err, "should throw error when 0 length array passed")
 	assert.Nil(t, schema, "should not have returned a schema for zero array")
 
-	schema, err = buildArraySchema(reflect.ValueOf([1]complex128{}), nil)
+	schema, err = buildArraySchema(reflect.ValueOf([1]complex128{}), nil, false)
 	_, expectedErr := GetSchema(reflect.TypeOf(complex128(1)), nil)
```

Comment:
and your getSchemas here are still in caps, is that right?

Reply:
It doesn't have an impact but since it now calls getSchema we probably should use getSchema incase GetSchema changes down the line